--- a/config.c	2025-02-13 18:27:37.349197294 -0500
+++ b/config.c	2025-02-13 18:28:49.815723797 -0500
@@ -4,7 +4,7 @@
 
 #include <ctype.h>
 #ifndef _MSC_VER
-#include <iniparser.h>
+#include <iniparser/iniparser.h>
 #endif
 #include <math.h>
 
@@ -750,13 +750,8 @@
     if (p->userEQ_keys > 0) {
         p->userEQ_enabled = 1;
         p->userEQ = (double *)calloc(p->userEQ_keys + 1, sizeof(double));
-#ifndef LEGACYINIPARSER
         const char *keys[p->userEQ_keys];
         iniparser_getseckeys(ini, "eq", keys);
-#endif
-#ifdef LEGACYINIPARSER
-        char **keys = iniparser_getseckeys(ini, "eq");
-#endif
         for (int sk = 0; sk < p->userEQ_keys; sk++) {
             p->userEQ[sk] = iniparser_getdouble(ini, keys[sk], 1);
         }
