diff -Naur a/NickvisionCavalier.Shared/org.nickvision.cavalier.metainfo.xml.in b/NickvisionCavalier.Shared/org.nickvision.cavalier.metainfo.xml.in
--- a/NickvisionCavalier.Shared/org.nickvision.cavalier.metainfo.xml.in	2023-07-21 11:48:44.565563047 +0530
+++ b/NickvisionCavalier.Shared/org.nickvision.cavalier.metainfo.xml.in	2023-07-21 12:14:33.584013059 +0530
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop">
   <id>org.nickvision.cavalier.desktop</id>
+  <icon type="stock">snap.cavalier.org.nickvision.cavalier</icon>
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>MIT</project_license>
   <name>Cavalier</name>
